docs/reference/migration/migrate_7_0/search.asciidoc

```diff
@@ -35,8 +35,15 @@ The Search API returns `400 - Bad request` while it would previously return
 *   number of filters in the adjacency matrix aggregation is too large
 
 
-==== Scroll queries cannot use the request_cache anymore
+==== Scroll queries cannot use the `request_cache` anymore
 
-Setting `request_cache:true` on a query that creates a scroll ('scroll=1m`)
+Setting `request_cache:true` on a query that creates a scroll (`scroll=1m`)
 has been deprecated in 6 and will now return a `400 - Bad request`.
 Scroll queries are not meant to be cached.
+
+==== Term Suggesters supported distance algorithms
+
+The following suggestion distance algorithms have been removed:
```

Comment:
This sounds a little more dramatic than it needs to: we haven't removed the algorithms, just the deprecated and non-preferred spellings. Could you reword more gently?

Reply:
Thanks @DaveCTurner! Can you have another look?